Gemfile.lock

```diff
@@ -1,32 +1,27 @@
 PATH
   remote: .
   specs:
-    codeclimate (0.69.0)
-      activesupport (>= 4.2.1, < 5.1)
-      codeclimate-yaml (~> 0.13.0)
+    codeclimate (0.66.0)
```

Comment:
hm, should this go back to `0.66` since we ended up getting up to `0.69` on master?  there is also the `VERSION` file on this branch which looks like it got updated to `0.69`.